_posts/2018/02/2018-02-28-lesson-infrastructure-subcommittee.md

```diff
@@ -0,0 +1,96 @@
+date: 2018-02-28
+time: "10:00:00"
+category: ["Community", "Lesson Infrastructure Subcommittee"]
+---
+
+On 14 February 2018 at 18:00UTC+0,
+the Lesson Infrastructure Subcommittee had their 2018 February meeting.
+This post will cover the topics discussed and their resolutions.
+
+## Software Carpentry and Data Carpentry merge
+
+Software Carpentry and Data Carpentry had merge.
+Their [Executive Council was elected](https://software-carpentry.org/blog/2017/12/executive-council-2018.html),
+they have a [new fiscal sponsor](https://software-carpentry.org/blog/2018/01/fiscal-sponsor-transition.html)
+and
+a [new logo](https://software-carpentry.org/blog/2018/02/our-new-logo.html).
+Also,
+they have [hired Dr. François Michonneau](http://www.datacarpentry.org/blog/curriculum-dev-scaling/) to lead curriculum development efforts.
+
+This all are very exciting news. And we even didn't finished February.
+
+## Windows Installer
+
+We will proceed to adopt Git for Windows v2.15.1,
+which includes nano,
+and drop our Windows installer.
+This should simplify our installation instructions
+and
+provide a better experience for our learners during the workshop.
+All the details of this changes are covered in [this document](https://github.com/carpentries/lesson-infrastructure/blob/master/liep/001-git-for-windows-nano.md)
+that summarise many pull requests and issues over the last 6 months.
+
+## Workshop Installation Instructions
+
+In January,
+a [blog post was published](https://software-carpentry.org/blog/2018/01/proposal-workshop-template.html)
+with a proposal to enhancement the process to customize the installation instructions at the workshop page.
+After a initial round of feedback,
+Raniere Silva,
+François Michonneau
+and
+Rémi Emonet
+will move the proof of concept included in the proposal forward to a feasibility prototype.
+The plan is to have the prototype ready in April
+and,
+if we decide to go forward,
+made it available to instructors in July
+so workshops in August could benefit for this.
+
+## Lesson Release 2018/01
+
+Christina Koch (for Software Carpentry)
+and Erin Becker (for Data Carpentry)
+will coordinate the next Lesson Release,
+planned for the middle of the year.
+
+
+## Maintainer survey on template
+
+As [announced on the Maintainers mailing list](http://lists.software-carpentry.org/pipermail/maintainers/2018-February/000453.html),
+François Michonneau is running a survey around our lesson template.
+We should hear the results soon.
+
+## Labels and Lessons
+
+If you don't know,
+[GitHub rolled out some improvements to labels](https://github.com/blog/2505-label-improvements-emoji-descriptions-and-more).
+This changes couldn't be in a better time.
+Erin Becker
+and
+François Michonneau
+are working to revamp the labels used in the Git repositories that host the lessons.
+The new set of labels will facilitate for instructors and learners to navigate on issues and pull requests.
```

Comment:
will facilitate the navigation around issues and pull requests for maintainers and contributors.